### PR TITLE
Clean up: custom coordinate space

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions 
- establish correct order within a property

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

File: schemas/non-atlas/customCoordinateSpace.schema.tpl.json [hunk 1]
```diff
@@ -18,10 +18,10 @@
     },    
     "axesOrigin": {
       "type": "array",        
+      "minItems": 2,      
       "maxItems": 3,    
-      "minItems": 2,
       "uniqueItems": true,
-      "_instruction": "Enter the origin of this custom coordinate space (central point where axes intersect; 2D: [x, y] or 3D:[x, y, z]).",
+      "_instruction": "Enter the origin (= the central point where all axes intersect) of this common coordinate space for two-dimensonal spaces as [x, y] or for three-dimensional space as [x, y, z].",
```
Comment (lzehl) [inline schemas/non-atlas/customCoordinateSpace.schema.tpl.json]: "Enter the origin (central point where all axes intersect) of this common coordinate space for two-dimensional spaces as [x, y] or for three-dimensional space as [x, y, z]."

**Review by lzehl — CHANGES_REQUESTED** [review]

**Review by lzehl — APPROVED** [review]
@aeidi89 I've updated the file in regard to my issue. Please double check and merge if you also approve now.

File: schemas/non-atlas/customCoordinateSpace.schema.tpl.json [hunk 2]
```diff
@@ -18,10 +18,10 @@
     },    
     "axesOrigin": {
       "type": "array",        
+      "minItems": 2,      
       "maxItems": 3,    
-      "minItems": 2,
       "uniqueItems": true,
-      "_instruction": "Enter the origin of this custom coordinate space (central point where axes intersect; 2D: [x, y] or 3D:[x, y, z]).",
+      "_instruction": "Enter the origin (central point where all axes intersect) of this common coordinate space for two-dimensional spaces as [x, y] or for three-dimensional space as [x, y, z].",
```
Comment (aeidi89) [inline schemas/non-atlas/customCoordinateSpace.schema.tpl.json]: As this is the custom coordinate space schema, shouldn't the instructions point to custom and not common coordinate space?

**Review by aeidi89 — APPROVED** [review]
I fixed a minor typo, changing 'common' to 'custom' in the instruction

**Pull request merged**